src/gui/torrentcontentmodel.cpp

```diff
@@ -88,6 +99,43 @@ namespace
             return QIcon(iconPixmap);
         }
     };
+#elif defined(Q_OS_MAC)
+    // There is a similar bug on macOS, to be reported to Qt
+    // https://github.com/qbittorrent/qBittorrent/pull/6156#issuecomment-316302615
+    class MacFileIconProvider: public UnifiedFileIconProvider
+    {
+    public:
+        using QFileIconProvider::icon;
+        QIcon icon(const QFileInfo &info) const override
+        {
+            const auto suffix = info.suffix();
```

Comment:
I think it is just better to give it a different name in such case. Fixed anyway.